scripts/apothecary/formulas/FreeImage/FreeImage.sh

```diff
@@ -10,20 +10,20 @@
 FORMULA_TYPES=( "osx" "vs" "win_cb" "ios" "android" )
 
 # define the version
-VER=3160 # 3.16.0
+VER=3170 # 3.16.0
```

Comment:
Good reason. We should get the other platformes compiled and up to date once this is merged.